manifests/manifest.COMMON.json

```diff
@@ -1,50 +1,40 @@
 {
-  "manifest_version": 2,
-  "name": "linkding extension",
-  "version": "1.7",
+  "manifest_version": 3,
+  "name": "Linkding",
+  "version": "2.0.0",
   "description": "Companion extension for the linkding bookmark service",
   "homepage_url": "https://github.com/sissbruecker/linkding-extension/",
-
   "icons": {
     "48": "icons/logo_48x48.png",
     "96": "icons/logo_96x96.png",
     "128": "icons/logo_128x128.png"
   },
-
-  "background": {
-    "scripts": ["build/background.js"]
-  },
-
   "omnibox": {
     "keyword": "ld"
   },
-
-  "browser_action": {
+  "action": {
     "default_icon": {
       "19": "icons/button_19x19.png",
       "32": "icons/button_32x32.png",
       "38": "icons/button_38x38.png"
     },
-    "default_title": "Add bookmark (Alt+Shift+L)",
+    "default_title": "Add bookmark",
```

Comment:
I only removed it since none of the other extensions I use that have shortcuts, add the shortcut to their title. It's not a trend I'm familiar with when it comes to browser extensions. All the same, I agree that the shortcut isn't surfaced anywhere else at the moment, so I'll let you make the call on whether it should stay or go, and we can always revisit this later.